app/controllers/home_controller.rb

```diff
@@ -0,0 +1,7 @@
+class HomeController < ApplicationController  
+  def index
+  end
+  
+  def app
+  end
+end
```

Comment:
The same controller created by @SerhiiBorozenets in CI-34/CI-35

Reply:
without it, I can't test the React settings + it is included in the React settings